### PR TITLE
[WC-1104] Add Document Viewer docs page

### PR DESCRIPTION
**Pull request by iobuhov**
This is documentation for new Document Viewer docs.

### CODE REVIEW

**Comment by ConnorLand** [comment]
Hi @iobuhov is this widget already released? Including the release date on new widget doc PRs helps me not merge them early.

**Comment by iobuhov** [comment]
Hi @ConnorLand, my apologies, I meant to add Future Release tag, but unfortunately if forgot to do so.
Answering your question - yes, this widget is not yet released.

File: content/en/docs/appstore/widgets/document-viewer.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+description: "Documentation for Document Viewer widget. Describes the usecase, features, and configuration of the Document Viewer widget which is available in the Mendix Marketplace."
+tags: ["marketplace", "marketplace component", "widget", "document", "viewer", "platform support", "pdf", "preview"]
+#If moving or renaming this doc file, implement a temporary redirect and let the respective team know they should update the URL in the product. See Mapping to Products for more details.
+---
+
+
+
+## 1 Introduction {#introduction}
+
+The Document Viewer widget allows you to preview document content.
```
Comment (ConnorLand) [inline content/en/docs/appstore/widgets/document-viewer.md:14]: Can you expand a little here on possible use cases?

File: content/en/docs/appstore/widgets/document-viewer.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+#### 2.2.3 Height Unit {#height-unit}
+
+Hight unit controls type of unit for measuring widget height. Possible values are:
+
+* **Percentage of width** — height is measured relative to widget width (use this mode to keep width and height in aspect ratio).
+* **Pixels** — use pixels as the height unit (a good default option for most cases).
+* **Percentage of parent** — height is measured relative to the parent's `height` property (this only works when the parent has the `height` CSS property).
+
+
+#### 2.2.4 Height {#height}
```
Comment (ConnorLand) [inline content/en/docs/appstore/widgets/document-viewer.md:55]: Nice draft. I understand that it's a basic widget, but are there any other "best practices", patterns, or pieces of configuration advice we can give? Might be nice in a final section.

Comment (iobuhov) [reply content/en/docs/appstore/widgets/document-viewer.md:55]: I'll try to think what we can add, but to be honest I have very little knowledge about "common" use cases for this widget. I'll try to do my best.

**Comment by iobuhov** [comment]
@ConnorLand  I think this PR can be closed as we stopped development of Document viewer

**Comment by ConnorLand** [comment]
> @ConnorLand I think this PR can be closed as we stopped development of Document viewer

Understood. I'll close it and follow up with you on this issue.

**Pull request closed without merging**